modules/gapi/src/executor/gstreamingexecutor.cpp

```diff
@@ -1397,17 +1403,28 @@ cv::gimpl::GStreamingExecutor::GStreamingExecutor(std::unique_ptr<ade::Graph> &&
                 // Initialize queues for every operation's input
                 ade::TypedGraph<DataQueue, DesyncSpecialCase> qgr(*m_island_graph);
                 bool is_desync_start = false;
+                bool drop_last = false;
```

Comment:
If `drop_last` is a parameter to `desync()` operation, ideally nobody but `desync()` itself should know about that.